new-filename.js

```diff
@@ -0,0 +1,3 @@
+if(foo === "bar") {
+  console.log('something new for real2')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.dev.codacy.org/gh/troubleshoot-codacy/no-suggestions/pullRequest?prid=30886)
```suggestion
  console.log("something new for real2")
```